lib/samples/apply_unique_value_renderer/apply_unique_value_renderer_sample.dart

```diff
@@ -27,54 +27,62 @@ class ApplyUniqueValueRendererSample extends StatefulWidget {
 
 class _ApplyUniqueValueRendererSampleState
     extends State<ApplyUniqueValueRendererSample> {
+  // create a controller for the map view.
   final _mapViewController = ArcGISMapView.createController();
 
   @override
   void initState() {
```

Comment:
An `onMapViewReady` function should be added and the `initState` code should be moved there.